cli/cmd/release_lint.go

```diff
@@ -127,10 +131,40 @@ func tarYAMLDir(yamlDir string) ([]byte, error) {
 		return nil, errors.Wrap(err, "failed to archive")
 	}
 
-	data, err := ioutil.ReadFile(archiveFile)
+	data, err := os.ReadFile(archiveFile)
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to read archive file")
 	}
 
 	return data, nil
 }
+
+func isHelmChartsOnly(yamlDir string) (bool, error) {
+	helmError := errors.New("helm error")
+	err := filepath.Walk(yamlDir, func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() {
+			return nil
+		}
+
+		_, err = loader.LoadFile(path)
+		if err != nil {
+			return helmError
+		}
+
+		return nil
+	})
+
+	if err == helmError {
+		return false, nil
+	}
```

Comment:
Do we want to return a `nil` error here?

Reply:
it's not an error.  it's the `false` result.

Reply:
```
_, err = loader.LoadFile(path)
```
We are basically using this call to see if this an Helm chart right? Is it possible returns other errors like "file not found"?  Additionally, do we need to wrap this helm error?

Reply:
If it can't be loaded then it's not a chart. There are no restrictions on what kinds of files can be passed in.